core/templates/dev/head/domain/statistics/LearnerAnswerInfoObjectFactory.ts

```diff
@@ -17,52 +17,70 @@
  * domain objects.
  */
 
-angular.module('oppia').factory('LearnerAnswerInfoObjectFactory', [
-  function() {
-    var LearnerAnswerInfo = function(learnerAnswerInfoId, answer,
-        answerDetails, createdOn) {
-      this._id = learnerAnswerInfoId;
-      this._answer = answer;
-      this._answerDetails = answerDetails;
-      this._createdOn = createdOn;
-    };
+import { Injectable } from '@angular/core';
+import { downgradeInjectable } from '@angular/upgrade/static';
 
-    LearnerAnswerInfo.prototype.getId = function() {
-      return this._id;
-    };
+export interface LearnerAnswerInfo {
```

Comment:
Please rename to: `LearnerAnswerInfoBackendDict`

Reply:
Hey Brian, could you please take a look at it again and let me know if it is still needed. I'll change it as per your request then.